package.json

```diff
@@ -58,15 +58,19 @@
   },
   "dependencies": {
     "3d-view": "^2.0.0",
-    "@plotly/d3-sankey": "^0.5.1",
     "alpha-shape": "^1.0.0",
     "array-range": "^1.0.1",
     "canvas-fit": "^1.5.0",
     "color-normalize": "^1.3.0",
     "convex-hull": "^1.0.3",
     "country-regex": "^1.1.0",
     "d3": "^3.5.12",
+    "d3-sankey": "git://github.com/antoinerg/d3-sankey.git#4f37ed8d3578b545a8569ecd74583f373768e900",
+    "d3-sankey-circular": "git://github.com/antoinerg/d3-sankey-circular.git#e5760737ef64afade7906bd018149bf1da080e2d",
```

Comment:
Minimal changes over latest d3-sankey-circular:

 https://github.com/tomshanley/d3-sankey-circular/compare/master...antoinerg:support-update

Reply:
Cool. We'll have to make a new (or move it?) to the `@plotly` organization.

Reply:
... unless you think your patches are general enough to hope that a PR would get merged over at https://github.com/tomshanley/d3-sankey-circular

Reply:
Mmm... I guess we have a shot at getting a PR merged in. Basically, `d3-sankey-circular` is missing the API function `update()` present in `d3-sankey`. This is useful for implementing interactivity/animations.

Reply:
> I guess we have a shot at getting a PR merged in

PR away please :rocket: !

Reply:
It got merged in pretty quickly! https://github.com/tomshanley/d3-sankey-circular/pull/28

It is not yet published on npmjs.com and I don't know if it will be. Is it OK to point to a git repo like I was doing? If it is, we should keep a copy of it in case the repo disappears. Let me know what you prefer!